src/lib/libast/vmalloc/vmbest.c

```diff
@@ -19,7 +19,7 @@
 *                     Phong Vo <phongvo@gmail.com>                     *
 *                                                                      *
 ***********************************************************************/
-#if defined(_UWIN) && defined(_BLD_ast)
+#if !defined(_BLD_ast)
```

Comment:
This is wrong. The enclosed code should not be compiled if `_UWIN` is not defined. It should therefore be removed rather than predicated on `_BLD_ast` not being defined. Notice the `a && b` form of the statement you are replacing.

Reply:
Have you built it that way ?
It doesn't build the way you describe.
This pattern will manifest a dozen times in the PR.

Reply:
> Have you built it that way ?

No, and it doesn't matter. Think about the condition you are modifying. If the condition is `a && b` and `a` can never be true then it does not matter whether or not `b` is true. The code conditioned on this statement will never be compiled. So it is incorrect to replace `a && b` with `!b`.